fastlane/lib/fastlane/lane_manager.rb

```diff
@@ -104,17 +104,31 @@ def self.print_table(actions)
     # @param platform: is probably nil, but user might have called `fastlane android`, and only wants to list those actions
     def self.choose_lane(ff, platform)
       loop do
-        UI.error "You must provide a lane to drive. Available lanes:"
-        available = ff.runner.available_lanes(platform)
-
+        UI.message "Welcome to fastlane! Here's what your app is setup to do:".white
+        available = ff.runner.lanes[platform].to_a
+        rows = []
         available.each_with_index do |lane, index|
-          UI.message "#{index + 1}) #{lane}"
+          row = [index + 1, lane.last.pretty_name, lane.last.description.first]
```

Comment:
This shouldn't be `.first` as far as I know, but `.join(" ")` or `.join("\n")` as a description can be multi-line. We definitely need to make sure the table design doesn't break when the line is too long